From c8a1960b165f06d198888d9a89e3edbe8645c5d5 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Tue, 14 May 2019 11:32:45 +0200
Subject: [PATCH 1115/1200] arm64: dts: cn9132: enable pcie2 on CP2

By mistake the cp2_pcie0 was enabled twice, while one of the entry was
aimed to reference cp2_pci2.

Change-Id: I8fc2dedfeaaa15cd93fdc5c94e3e740d152c0afe
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/8951
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Nadav Haklai <Nadav.Haklai@cavium.com>
---
 arch/arm/dts/cn9132-db-A.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/cn9132-db-A.dts b/arch/arm/dts/cn9132-db-A.dts
index 24a74c2ba0..a8930d46d8 100644
--- a/arch/arm/dts/cn9132-db-A.dts
+++ b/arch/arm/dts/cn9132-db-A.dts
@@ -62,7 +62,7 @@
 };
 
 /* SLM-1521-V2, CON8 */
-&cp2_pcie0 {
+&cp2_pcie2 {
 	num-lanes = <1>;
 	status = "okay";
 };
-- 
2.22.0

